client/tx/tx_test.go

```diff
@@ -40,7 +40,7 @@ type mockContext struct {
 	wantErr bool
 }
 
-func (m mockContext) Invoke(grpcCtx context.Context, method string, req, reply interface{}, opts ...grpc.CallOption) (err error) {
+func (m mockContext) Invoke(_ context.Context, _ string, _, reply interface{}, _ ...grpc.CallOption) (err error) {
```

Comment:
If no parameters are used, may as well omit all the parameter names: `(type1, type2)` instead of `(_ type1, _ type2)`. But it's a test function, so not high importance.